### ISSUE TITLE
Select Python Interpreter Issue

### ISSUE THREAD

**Issue by qmfinger** [microsoft/vscode-docs#4420]
Hello,

I have done a fresh install of VSCode/Python/Git to start working from home.  I cloned my project down and I am using a venv so I don't need to keep trying to match lib files between work and home.  I go to select my Python Interpreter located in venv\Scripts\python.exe and nothing changes.  It acts as if nothing was selected and still tell me to Select Python Interpreter.  I confirm this by going into the terminal and activate the venv and then type python.  I get " No Python at 'C:\Users\XxQxX\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.8_qbz5n2kfra8p0\python.exe' which is not the location that I selected through the command pallet.

Any assistance would greatly be appreciated.
-Q

**Comment by JoshuaPartlow** [comment]
This issue seems similiar to the one reported in https://github.com/microsoft/vscode-docs/issues/4414 @luabud is there any known issue around this? Should these two issues instead be on the extension repo?

**Comment by luabud** [comment]
Hey @qmfinger! Would you mind filing an issue at the Python extension repo, providing more information so we can investigate what is going on? https://github.com/microsoft/vscode-python/issues/new?assignees=&labels=classify%2C+type-bug&template=2_bug_report.md

In particular we're interested in the contents of your Python output channel:

![image](https://user-images.githubusercontent.com/45497113/112698908-f76efb80-8e47-11eb-947a-961d4df81b3d.png)

**Comment by qmfinger** [comment]
@luabud & @JoshuaPartlow , Thank you for reaching out to assist me in this.  I actually got it figured out.  I didn't realize that when you create a venv with python that it would create a config file in the venv folder.  Found out that the "home" variable pointed to my work pc's python file and even if I used the command pallet it wouldn't change that.  Further into my research I realized it might be bad practice to store my venv in my git repository and it would be better to use a pip freeze command to store my requirements in my git repository instead.  

That being said, I did run into an interesting bug.  If you change the home variable to point at the python.exe that is created in the python -m venv venv command it will actually create an infinite loop when you try to run the python command from the terminal.  If you have your task manager open and look at the performance it will slowly max out your cpu and ram capacity.  Had to restart my machine once.
